engine/resource/src/resource_mounts.cpp

```diff
@@ -271,13 +282,27 @@ dmResource::Result LoadMounts(HContext ctx, const char* app_support_path)
         return result;
     }
 
+    char app_home_path[1024];
+    dmSys::Result app_path_result = dmSys::GetHomePath(app_home_path, sizeof(app_home_path));
```

Comment:
To be honest, I were thinking we were going to use GetApplicationSupportPath() for this?
What's the benefit of the new function?

Reply:
Unfortunately, we can't use `GetApplicationSupportPath` because this is just one of  the subfolders:
`/var/mobile/Containers/Data/Application/{app-uid?}/*subfolder here*`
and there are many subfolders may be used. We need to replace the main path before `{app-uid?}`